mathics/builtin/arithfns/basic.py

```diff
@@ -520,6 +528,8 @@ class Power(BinaryOperator, _MPMathFunction):
     rules = {
         "Power[]": "1",
         "Power[x_]": "x",
+        "Power[I,-1]": "-I",
```

Comment:
In master, this rule is not needed.: `Power[I, -1] ` returns `-I`. 

In this branch though I am seeing `(-I) ^ (-1 / 2)`.  Which one matches WMA. And why the discrepancy?

Reply:
I do not consider your different implementation as a good thing but a bad thing. Given the choice of relying on Sympy code or mmatera code we should go with Sympy. It is more used, more maintained ,and generally more well thought out and more complete. 

Our job should be to figure out how to get WMA thinks done in Sympy when it can.